frontend/src/components/public_group/PublicGroupHero.js

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 import PublicTopBarV2 from '../../containers/PublicTopBarV2';
+import formatCurrency from '../../lib/format_currency';
+import filterCollection from '../../lib/filter_collection';
 
 export default class PublicGroupHero extends React.Component {
```

Comment:
It enforces not storing any state in React but rather in Redux, and will be subject to performance improvements in future react versions as the functional component can be flagged "pure". There is also probably less overhead to begin with, as there is no lifecycle management. But it's still an early feature. 

https://facebook.github.io/react/blog/2015/09/10/react-v0.14-rc1.html#stateless-function-components

> This pattern is designed to encourage the creation of these simple components that should comprise large portions of your apps. In the future, we’ll also be able to make performance optimizations specific to these components by avoiding unnecessary checks and memory allocations.